aunt-leahs-app/src/components/TableComponent.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../styles.css';
+import { Table, TableBody, TableCell, Paper, TableHead, TableRow } from '@material-ui/core';
+
+// REQUIRES: data prop has to be array of objects where each object key represents the name of the field in camelcase.
```

Comment:
This shouldn't be a requirement to the input. You should instead have an array for headers and table data separately.

Reply:
For now, since we don't know the schema of database I'd like to stick to this implementation.
Change might be necessary later on but I need the table component for other pages for now.